package.json

```diff
@@ -7,7 +7,7 @@
   },
   "devDependencies": {
     "commander": "^2.20.3",
-    "lerna": "^3.22.1",
```

Comment:
This is a big major bump, is there any potential breaking change that would break our workflow?

Reply:
well, i'm testing it right now... 🤞

Reply:
seems like everything still works!